repl/appengine/README.md

```diff
@@ -0,0 +1,31 @@
+# CEL REPL Web
+Provides a simple web interface for working with the CEL REPL.
+
+The two main components are 1) a Go application that implements a service
+wrapping the REPL and serving static web content 2) an angular based web page
+that provides an interface around the API.
+
+The REPL service is stateless -- it initializes a new REPL instance and applies
+the requested commands in order on each request.
+
+## Development
+
+To run the application in development, run the angular CLI builder in watch mode
+and run the Go server as follows:
+
+```
+# from the `repl/appengine/web` directory:
+ng build --watch
```

Comment:
Since we're including npm scripts in `package.json`, we can change this to be ran without installing the Angular CLI:

```suggestion
npm run watch
```